test/optimization/test_warm_start_qaoa.py

```diff
@@ -0,0 +1,61 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2020.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+""" Test warm start QAOA optimizer. """
+
+from test.optimization import QiskitOptimizationTestCase
+
+import numpy as np
+from qiskit import BasicAer
+from qiskit.aqua.algorithms import QAOA
+from qiskit.optimization.algorithms.goemans_williamson_optimizer import GoemansWilliamsonOptimizer
+from qiskit.optimization.algorithms.warm_start_qaoa_optimizer import WarmStartQAOAOptimizer, \
+    MeanAggregator
+from qiskit.optimization.applications.ising.max_cut import max_cut_qp
+
+
+class TestWarmStartQAOAOptimizer(QiskitOptimizationTestCase):
```

Comment:
I think we can add a couple of tests where a portfolio problem is used (or something equivalent). This would allow us to test the relaxation to a continuous QP. Perhaps something of the form
```
1. setup a simple QUBO
2. relax to QP and test that we get the right solution
3. use that to warm-start a QAOA
4. solve the QAOA with a fixed initial (beta, gamma) and solver.
5. Test that we get the correct result
```

Reply:
Added a couple of tests more. Feel free to add even more.